lib/Client.php

```diff
@@ -22,17 +23,36 @@ class Client
     private $sentryClient;
 
     /**
-     * @param GuzzleClient $client
+     * @param \GuzzleHttp\Client $client
      * @param string $apiKey
+     * @param \Raven_Client $sentryClient
+     * @param int $timeout
```

Comment:
@ricardotulio please declare this parameter as optional using the docblock syntax, as below:

```php
/**
 * @param int|null $timeout
 */
```